deployment/keystone/deploy_test.go

```diff
@@ -26,6 +28,7 @@ import (
 
 func TestDeploy(t *testing.T) {
 	lggr := logger.Test(t)
+	ctx := testcontext.Get(t)
```

Comment:
No need to pull in CTF here:
```suggestion
	ctx := tests.Context(t)
```

Reply:
In this case, I was pulling it in because elsewhere in the integration test environment we use that one. I don't have a strong opinion, but didn't want to deviate.